### PR TITLE
Remove V1 for SA360

### PR DESCRIPTION
**Pull request by smcjones**
SA360's old API has been sunset. We don't need this cluttering our codebase anymore as it's completely unusable for clients. We move "V2" to be the default now.

### CODE REVIEW

**Comment by smcjones** [comment]
This should be reviewed AFTER https://github.com/google-marketing-solutions/margin-protection/pull/13

**Review by anaesqueda — CHANGES_REQUESTED** [review]
Hi Sean, looks like there are some merge comments in this PR, could you please remove them? Thanks!

File: sa360/src/api.ts [hunk 1]
```diff
@@ -19,8 +19,111 @@
  * @fileoverview DAO for the SA360 Reporting API
  */
 
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (smcjones) [reply sa360/src/api.ts]: HI @anaesqueda build was failing due to this and it should already be fixed.

**Review by anaesqueda — APPROVED** [review]
LGTM

**Pull request merged**